### PR TITLE
Fixes #15115: Add a variable documentation page

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15115

### CODE REVIEW

File: src/reference/modules/reference/pages/variables.adoc [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+----
+// Call to a String or Iterator variable
+${<prefix>.<variable name>}
+
+// Call to a key in a Dict variable
+${<prefix>.<variable dict name>[key][sub-key]}
+
+----
+
```
Comment (amousset) [inline src/reference/modules/reference/pages/variables.adoc]: You can add a note about the `$(...)` alternative but deprecated syntax on Unix agents.

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

File: src/reference/modules/reference/pages/variables.adoc (hunk 1 above)
Comment (ncharles) [reply src/reference/modules/reference/pages/variables.adoc]: it is on line 10 ?

**Pull request merged**